app/javascript/components/components/Header.js

```diff
@@ -32,7 +33,34 @@ class Header extends Component {
             </NavItem>
           }
         </Nav>
+
+        <div>
+          <Helmet>
+            <style>{'body { background-color:#2cd5eb; }'}</style>
+          </Helmet>
+
+        </div>
+
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+          <a class="navbar-brand" href="/">Home</a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav mr-auto">
+              <li class="nav-item active">
+                <a class="nav-link" href="/apartmentindex">See All Apartments <span class="sr-only"></span></a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="/apartmentnew">Upload an Apartment Listing</a>
```

Comment:
Keep in mind that you will likely want this to conditionally render if a user is logged in.